src/cloud/trouble/error-codes.md

```diff
@@ -0,0 +1,115 @@
+---
+group: cloud-guide
+title: Error message reference for ece-tools
+---
+
+{% assign errors = site.data.cloud-error-messages %}
+
+This {{site.data.var.ct}} error message reference provides information to resolve errors that occur during the {{site.data.var.ece}} deployment process.
+
+Error messages are categorized by deployment stage–build, deploy, and post-deploy. Each section provides a list of associated errors with the following information for each error:
+
+-  **Error code**:  The Magento-assigned identifier for the error message
+-  **Step**:  Indicates the step in the deployment scenario where the error can occur. If the _Step_ column is blank, the error is a general error that can occur in multiple steps, or during pre-processing operations. See [Scenario-based deployment]({{site.baseurl}}/cloud/deploy/scenario-based-deployment.html) for more information about the build, deploy, and post-deploy steps.
+-  **Error description**: A short phrase that summarizes the cause of the error
+-  **Suggested action**: Provides guidance to troubleshoot and resolve the error
+
+## Build stage
+
+{: .error-table}
+| Error code | Build step | Error description | Suggested action |
+| ---------- | ---------- | ----------------- | ---------------- |
+| 2 | | Cannot write to the `./app/etc/env.php` file | {{site.data.cloud-error-messages.ENV_PHP_IS_NOT_WRITABLE}}|
+| 3 | | Configuration isn't defined in the `schema.yaml` file | {{site.data.cloud-error-messages.CONFIG_NOT_DEFINED}}|
+| 4 | | Failed to parse the `.magento.env.yaml` file | {{site.data.cloud-error-messages.CONFIG_PARSE_FAILED}}|
+| 5 | | Unable to read the `.magento.env.yaml` file | {{site.data.cloud-error-messages.CONFIG_UNABLE_TO_READ}}|
+| 6 | | Unable to read the `.schema.yaml` file | |
+| 7 | refresh-modules | Cannot write to the `./app/etc/config.php` file |{{site.data.cloud-error-messages.CONFIG_PHP_IS_NOT_WRITABLE}} |
```

Comment:
I do like the tables. It really helps with self-sufficiency when you have reference material like this. I do think the Build step is easier to see if it is formatted as inline code. Give it a try and see what you think.

```suggestion
| 7 | `refresh-modules` | Cannot write to the `./app/etc/config.php` file |{{site.data.cloud-error-messages.CONFIG_PHP_IS_NOT_WRITABLE}} |
```

Reply:
@hguthrie  Devs added some additional updates. I'll add your suggestions in another PR. Thanks for the review.